tests/pfcwd/test_pfcwd_cli.py

```diff
@@ -16,6 +16,7 @@
 from tests.common.utilities import wait_until
 from tests.common.cisco_data import is_cisco_device
 from tests.common import config_reload
+from tests.common.devices.eos import EosHost
```

Comment:
We ensure the neighbor is eos, is there any possible that the neighbor is not eos?